src/Parser/Core/Modules/DeathTracker.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { formatMilliseconds } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const debug = true;
+
+class DeathTracker extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  deathTimestamp = 0;
+  resurrectTimestamp = 0;
+  timeDead = 0;
+  castCount = 0;
+  isAlive = true;
+
+  on_toPlayer_death(event) {
+    this.deathTimestamp = this.owner.currentTimestamp;
+    if (debug) {console.log("Player Died @ " + formatMilliseconds(event.timestamp - this.owner.fight.start_time));}
```

Comment:
This could be shorter: `debug && console.log(...);` or without the curly brackets.

The strict code style requires if-statements with curly brackets to be multiline.